<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Don't Panic</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>620</width>
     <height>620</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#quitButton {
    background-color: #f44;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 30px;
    min-width: 2em;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>141</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="scoreLabel">
      <property name="styleSheet">
       <string notr="true">#scoreLabel{
font-weight: bold;
font-size: 30px;
}</string>
      </property>
      <property name="text">
       <string>Score</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="styleSheet">
       <string notr="true">#lcdNumber{
	background-color: black;
}	</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>131</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelAvailableMoves">
      <property name="styleSheet">
       <string notr="true">#labelAvailableMoves{
font-weight: bold;
font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>Available moves</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdAvailableMoves">
      <property name="styleSheet">
       <string notr="true">#lcdAvailableMoves{
	background-color: black;
}	</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>scoreLabel</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>scoreLabel</zorder>
  <zorder>quitButton</zorder>
  <zorder>scoreLabel</zorder>
  <zorder>lcdAvailableMoves</zorder>
  <zorder>labelAvailableMoves</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
